mypy/main.py

```diff
@@ -179,6 +179,7 @@ def process_options(args: List[str],
 
     strict_flag_names = []  # type: List[str]
     strict_flag_assignments = []  # type: List[Tuple[str, bool]]
+    valid_disallow_any_options = {'unimported'}
```

Comment:
More importantly, this should be a list instead of a set, because it's displayed to our users in a couple places.  This is important for two reasons:  1) We don't want a nondeterministic `--help` output.  2) We're going to want to specify the options in a particular order that we think makes the most sense to read them in.

A set is canonically the "right" data structure to use here, because you're mainly checking for membership.  However, it's so small that it doesn't matter (especially because it's not going to be run in a tight loop or anything).